containers/pax-exam-container-forked/pom.xml

```diff
@@ -172,6 +172,21 @@
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <environmentVariables>
```

Comment:
Make use of [System Stubs](https://github.com/webcompere/system-stubs) to test with environment variables.